docs/policy/repo-layout-migration-2025.md

```diff
@@ -0,0 +1,45 @@
+OSG Software Repo Layout Migration
+==================================
+
+11 December 2024
+
+On Monday, January 6th, 2025, the OSG Software Team will be upgrading the server that hosts the OSG Software repos
+(<https://repo.osg-htc.org> a.k.a. <https://repo.opensciencegrid.org>),
+which will result in changes to the directory layout for the OSG 23 and OSG 24 YUM repos.
+
+Users should not be affected since the URLs of the repos themselves will not change.
+However, mirrors will have to take some action to avoid rsyncing those repos from scratch.
```

Comment:
```suggestion
However, mirror administrators will have to take some action to avoid rsyncing those repos from scratch.
```

AI has gone too far if the mirror services are making the adjustments themselves